src/device.rs

```diff
@@ -317,10 +319,10 @@ impl<B: UsbBus> UsbDevice<'_, B> {
                 }
 
                 for cls in classes.iter_mut() {
-                    cls.poll();
```

Comment:
This introduces a subtle change in behaviour that maybe should be documented: if there are multiple classes and an earlier one errors out, the later classes won't be polled.

However, looking at the current implementation and docstring, it's not explicit that polling the device (sometimes... unclear if that's correct) results in the classes being polled.